### PR TITLE
basic tests for df 

### PR DESCRIPTION
**Pull request by clewis7**
- [x] `create_df()`
- [x] `load_df()`
- [x] `set_parent_raw_data_path()`
- [x] `get_parent_raw_data_path()`
- [x] `add_item()`
    - [x] all animals
    - [x] all sessions for a single animal
    - [x] single session, all trials
    - [x] single session, single trial
- [x] `remove_item()`
    - [x] by row_ix
    - [x] all trials for a given animal
    - [x] all trials for a given animal session 
    - [x] single session, single trial

Note: will eventually have to revamp thing when it is no longer combined movie but rather front and side being stored for each trial

### CODE REVIEW

**Comment by clewis7** [comment]
https://pandas.pydata.org/docs/reference/api/pandas.testing.assert_frame_equal.html

should use this for ground truths

File: .gitignore [hunk 1]
```diff
@@ -22,4 +22,9 @@ share/python-wheels/
 *.egg-info/
 .installed.cfg
 *.egg
-MANIFEST
\ No newline at end of file
+MANIFEST
```
Comment (kushalkolar) [inline .gitignore:25]: you usually want to keep the MANIFEST file, why is it in .gitignore

File: animal_soup/batch_utils.py [hunk 2]
```diff
@@ -8,7 +8,12 @@
 CURRENT_DF_PATH: Path = None  # only one df at a time
 PARENT_DATA_PATH: Path = None
 
-DATAFRAME_COLUMNS = ["animal_id", "session_id", "ethograms", "type", "notes"]
+DATAFRAME_COLUMNS = ["animal_id",
```
Comment (kushalkolar) [inline animal_soup/batch_utils.py:11]: I think this style looks better, you could run `black` on the codebase?

```python3
l = [
    a,
    b,
]
```

**Comment by kushalkolar** [comment]
This is probably for a bit later, but are there any tests you can think of that can help make sure the ethogram you're currently viewing is from the array that is highlighted and not from another video? I think we ran into that issue quite a bit.

**Comment by clewis7** [comment]
> This is probably for a bit later, but are there any tests you can think of that can help make sure the ethogram you're currently viewing is from the array that is highlighted and not from another video? I think we ran into that issue quite a bit.

maybe adding uuid as a column? would help with preventing duplicates as well? idk, need to think about this one

**Pull request merged**